src/interface/src/app/plan/plan-map/plan-map.component.ts

```diff
@@ -36,6 +36,7 @@ export class PlanMapComponent implements OnInit, AfterViewInit, OnDestroy {
   @Input() mapId?: string;
   /** The amount of padding in the top left corner when the map fits the plan boundaries. */
   @Input() mapPadding: L.PointTuple = [0, 0]; // [left, top]
+  @Input() configPage: boolean = false;
```

Comment:
this is only for the attributionControl? I think we should name this like `showAttribution`, `showAttribuitonOnMap` or something like that instead

Reply:
It also controls if the zoom controls are added, though I agree the name could be better.  Would showZoomAndAttribution be too wordy?